app/jinja_filters.py

```diff
@@ -408,10 +408,10 @@ def __init__(  # noqa: C901, R0912  pylint: disable=too-complex,too-many-branche
             and "label" in answer
             and answer["label"]
         ):
-            self.title = answer["label"]
+            self.itemTitle = answer["label"]
             self.titleAttributes = {"data-qa": answer["id"] + "-label"}
         else:
-            self.title = question["title"]
+            self.itemTitle = question["title"]
```

Comment:
If `title` has been renamed `itemTitle`, shouldn't `titleAttributes` be renamed `itemTitleAttributes`?